### PR TITLE
Remove rbac policies from JWT token

### PR DESCRIPTION
**Pull request by davidjiglesias**
Hello team,

This PR adds a change to JWT token to eliminate full policies information. The new JWT token only shows the user, the roles and the rbac_mode, thus making the JWT token much shorter.

Regards,

David J. Iglesias

### CODE REVIEW

**Review by AdriiiPRodri — COMMENTED** [review]
Unit tests are not updated, currently it should fail

File: framework/wazuh/rbac/preprocessor.py [hunk 1]
```diff
@@ -86,7 +86,22 @@ def get_optimize_dict(self):
         return self.odict
 
 
-def optimize_resources(auth_context=None, user_id=None):
+def optimize_resources(roles=None):
+    """This function preprocess the policies of the user for a more easy treatment in the decorator of the RBAC
+
+    :param roles: Authorization context of the current user
+    :param user_id: Username of the current user
```
Comment (AdriiiPRodri) [inline framework/wazuh/rbac/preprocessor.py:93]: Remove this param

**Review by AdriiiPRodri — CHANGES_REQUESTED** [review]

File: framework/wazuh/rbac/auth_context.py [hunk 2]
```diff
@@ -308,9 +308,15 @@ def run_auth_context(self):
 
         return user_roles_policies
 
+    def run_auth_context_roles(self):
+        """This function will return the roles of aun user matching the authorization context"""
```
Comment (vicferpoy) [inline framework/wazuh/rbac/auth_context.py:312]:
```suggestion
        """This function will return the roles of a user matching the authorization context"""
```

**Review by vicferpoy — CHANGES_REQUESTED** [review]

File: api/api/authentication.py [hunk 3]
```diff
@@ -4,14 +4,14 @@
...
-import copy
 import logging
 import os
-from secrets import token_urlsafe
 from shutil import chown
 from time import time
 
+import copy
 from jose import JWTError, jwt
+from secrets import token_urlsafe
```
[the remark this answers is not in the document]
Comment (davidjiglesias) [reply api/api/authentication.py:14]: Done

**Review by vicferpoy — APPROVED** [review]
LGTM!

**Review by AdriiiPRodri — APPROVED** [review]

**Pull request merged**